e2e-tests/test/evm-apis.test.ts

```diff
@@ -56,13 +56,13 @@ describe("evm_setNextBlockTimestamp", function () {
     const userWallet = Wallet.createRandom().connect(provider);
 
     // Act
-    await provider.send("evm_setNextBlockTimestamp", [expectedTimestamp]);
+    await provider.send("evm_setNextBlockTimestamp", [expectedTimestamp.toString(16)]);
 
     await wallet.sendTransaction({
       to: userWallet.address,
       value: ethers.utils.parseEther("0.1"),
     });
-    expectedTimestamp += 2; // New transaction will add two blocks
+    expectedTimestamp += 1; // New transaction will add two blocks
```

Comment:
```suggestion
    expectedTimestamp += 1; // After tx is executed current timestamp will be increased by 1
```

Reply:
I'd rather propose "every transaction adds a block", or something like that...

Reply:
> Why? Not clear thing is that current block timestamp will be increased after tx is included into block.

Oh, that wasn't clear to me in your comment... :-) So, what about "After executing a transaction, the node puts it into a block and increases its current timestamp"?

Reply:
good

Reply:
OK, it's there.